dev/io.openliberty.microprofile.openapi.2.0.internal/test/io/openliberty/microprofile/openapi20/test/merge/clashing-path-merged.yaml

```diff
@@ -0,0 +1,120 @@
+openapi: 3.0.3
+info:
+  title: "Merged documentation"
+  version: "1.0"
+paths:
+  "/foo/foo":
```

Comment:
The path has had the context root added to the front of it. The context root will also be removed from the `servers` block so that the final URL ends up being the same.

Reply:
In this test, there's no severs in the source document so it would be generated by the servlet, which would work out the host and port and then append `openApiProvider.getApplicationPath()` which would return the context root.

However, when we merge documents, the resulting provider returns `null` from `getApplicationPath()` and so nothing is appended to the server URL in the final document.